### ISSUE TITLE
Fine-tuning by Transfer learning

### ISSUE THREAD

**Issue by lhy0322** [jerryji1993/DNABERT#32]
Hi, I want to realize the transfer learning from big data set to small data set on DNABERT，
First of all, I run the fine-tuning with big data sets，get a global model.
Take this model parameter as a starting point, and continue to fine tune the small data set.
The following log appears：
     06/11/2021 22:21:21 - INFO - __main__ -     Continuing training from checkpoint, will skip to saved global_step
     06/11/2021 22:21:21 - INFO - __main__ -     Continuing training from epoch 119
     06/11/2021 22:21:21 - INFO - __main__ -     Continuing training from global step 20000
     06/11/2021 22:21:21 - INFO - __main__ -     Will skip the first 8 steps in the first epoch
     Epoch: 0it [00:00, ?it/s]
     06/11/2021 22:21:21 - INFO - __main__ -    global_step = 20000, average loss = 0.0
     06/11/2021 22:21:21 - INFO - __main__ -   Saving model checkpoint to ./ft/out

It doesn't seem to be fine-tuning on small data sets.
Why does this happen?

**Comment by woreom** [comment]
so did you figure out the problem?

**Comment by lhy0322** [comment]
> so did you figure out the problem?

If there is [Optimizer.pt] in the directory of the pre-trained model,  DNABERT will automatically load it. The training will starts at the recorded checkpoint, so the training fails.  If you delete the [Optimizer.pt] in the directory, will be able to train successfully.
